uber/templates/preregistration/paid_preregistrations.html

```diff
@@ -50,22 +49,23 @@
 <p>The following preregistrations have been made from this computer:</p>
 
 <table class="table footable">
-  <tr><td>Legal Name</td><td>Badge Name</td><td>Badge Number</td><td>Confirmation Link</td></tr>
```

Comment:
Agreed.  That's one of the things on my mental TODO list which I plan on making a ticket for.  Another example customization I expect we'll need is what columns appear on the attendee and group overview pages.  Some events won't care about age at all, while others might want a custom field to be displayed which doesn't even exist in core Uber.  So that sort of thing definitely needs to be configurable as well.  For now I just went with the simplest thing, but these are examples of other places we'll want a predefined strategy for overriding.